Initializing your Skywallet.md

```diff
@@ -0,0 +1,115 @@
+# Initializing your Skywallet
+
+On successfully ensuring the genuineness of your Skywallet, you can begin with the initialization of it.
+
+## Step 1 :
+
+The first step in initializing your Skywallet is to install the Skycoin Desktop Wallet.
+
+As the Skycoin Desktop Wallet is used for the interaction between the Skywallet and the User.
+
+Please make sure to download the latest compatible version from the Skycoin [Downloads](https://www.skycoin.net/downloads/) Page.
+
+<If you are a Linux user then please make sure to run this script>
+
+If you already have the latest desktop wallet installed and configured, then you can skip this step.
+
+## Step 2:
+
+Connect your Skywallet to the USB port.
+
+## Step 3:
+
+Once you have connected the Skywallet to the USB port, open the Skycoin Desktop Wallet.
+
+If it is the first time you are using your Skywallet, please make sure to go through the following section of the user manual,(Checking the Genuineness of your Skywallet)[https://github.com/skycoin/hardware-wallet/wiki/How-to-check-whether-device-is-genuine], to ensure that you are using a genuine Skywallet.
+
+The instructions for a desktop wallet which is used for the first time and an already configured desktop wallet are different.
+
+### Step 3.1: First Time Use of a Desktop Wallet
+
+If it is for the first time you are using the desktop wallet then you will see the following screen :
+
+<Screenshot of a Desktop Wallet which is used for the first time>
+
+To configure your Skywallet, proceed to click on **“Using a hardware wallet?”** button.
+
+<Screenshot of a Desktop Wallet highlighting “Using a hardware wallet?” button>
+
+On pressing this button, the display screen on Skywallet will show a welcome message:
+
+<Reusing the Picture showing the Skywallet display with the welcome message>
+
+In order to configure your Skywallet for the first time, you have to install the firmware for the Skywallet, the same is explained in step #4.
```

Comment:
as explained in step #4. Check how you can link from this to chapter 4 in Markdown please.